helix-core/src/main/java/org/apache/helix/controller/stages/BestPossibleStateCalcStage.java

```diff
@@ -257,6 +257,10 @@ private boolean computeResourceBestPossibleState(ClusterEvent event, ResourceCon
 
         // Check if calculation is done successfully
         return checkBestPossibleStateCalculation(idealState);
+      } catch (HelixException e){
```

Comment:
Does Helix Style template not handle space in this case? If not, should improve the template.